docs/source/topics/proc_starting-monitoring-alerting-telemetry.adoc

```diff
@@ -11,7 +11,7 @@ Telemetry functionality is responsible for listing your cluster in the link:http
 For more information, see link:{crc-gsg-url}#accessing-the-openshift-cluster-with-oc_gsg[Accessing the OpenShift cluster with `oc`].
 * You must log in through [command]`oc` as the `kubeadmin` user.
 * You must assign additional memory to the {prod} virtual machine.
-At least 12 GiB of memory (a value of `12288`) is recommended.
```

Comment:
@sradco Are you sure putting this much memory needed just for enable monitoring and telemetry because we did test it with ~14GB without any issue.

Reply:
Updated to 14GB.